src/devices/ExplorerHat/Motorization/Motor.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.ExplorerHat.Motorization
+{
+    /// <summary>
+    /// Represent one of the onboard motors
+    /// </summary>
+    public class Motor : IDisposable
```

Comment:
IMO you should use DCMotor directly here since it doesn't seem like there would be any issues with that. If there is anything lacking in the DCMotor please file an issue instead of creating similar class